pages/all_events_en.vue

```diff
@@ -139,10 +142,11 @@
                     this.data.push(obj);
                 })
 
-                this.data = this.data.sort((a,b) => { return new Date(b.date) - new Date(a.date) });
-
                 this.dataUpcoming = this.filterData(this.data, true);
                 this.dataPast = this.filterData(this.data, false);
+
+                this.dataUpcoming = this.sortDatesChronologically(this.dataUpcoming);
```

Comment:
```js
this.dataUpcoming = this.filterData(this.data, true).sort(this.sortDatesChronologically);
this.dataPast = this.filterData(this.data, false).sort(this.sortDatesReverseChronologically);
```

No need for the reassignment. You should be able to pass in the function to `Array.sort`. But this is up to you